python_modules/dagster/dagster/_core/execution/asset_backfill.py

```diff
@@ -747,9 +752,20 @@ def execute_asset_backfill_iteration_inner(
     if request_roots:
         initial_candidates.update(asset_backfill_data.get_target_root_asset_partitions())
 
-        next_latest_storage_id = instance_queryer.get_latest_storage_id(
-            DagsterEventType.ASSET_MATERIALIZATION
+        next_latest_storage_id = None
```

Comment:
I think it's worth making sure the `CachingInstanceQueryer` has some method that does this, to avoid enlarging this already-large function.